src/tests/ftest/util/mpio_utils.py

```diff
@@ -115,44 +109,44 @@ def run_llnl_mpi4py_hdf5(self, basepath, hostfile, pool_uuid, test_repo,
         """
         print("self.mpichinstall: {}".format(self.mpichinstall))
         # environment variables only to be set on client node
-        env_variables = {"CRT_ATTACH_INFO_PATH": "{}/install/tmp/"\
-                             .format(basepath),
-                         "MPI_LIB": '',
-                         "MPIO_USER_PATH": "daos:",
-                         "DAOS_POOL": "{}".format(pool_uuid),
-                         "DAOS_SVCL": "{}".format(0),
-                         "HDF5_PARAPREFIX": "daos:"}
+        env = {
+            "CRT_ATTACH_INFO_PATH": "{}/install/tmp/".format(basepath),
+            "MPIO_USER_PATH": "daos:",
+            "DAOS_POOL": "{}".format(pool_uuid),
+            "DAOS_SVCL": "{}".format(0),
+            "HDF5_PARAPREFIX": "daos:"
+        }
+        mpirun = os.path.join(self.mpichinstall, "bin", "mpirun")
         # setting attributes
-        cd_cmd = 'cd ' + test_repo
+        cmd = exports_cmd(env)
```

Comment:
Same suggestion as above to use the `EnvironmentVariables` class

Reply:
Fixed.